<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(89, 122, 170, 255), stop:1 rgba(255, 255, 255, 255));}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,7">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="sidebar" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#centralwidget{
background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(89, 122, 170, 255), stop:1 rgba(255, 255, 255, 255));}</string>
      </property>
      <widget class="QPushButton" name="btnProfileSidebar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(40, 57, 83); 
color: white;
font: 16pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/settings.png</normaloff>images/settings.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btnExitSidebar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>400</y>
         <width>100</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(74, 0, 0);
color: white;
font: 16pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/exit.png</normaloff>images/exit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btnStatsSidebar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>79</y>
         <width>100</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(40, 57, 83); 
color: white;
font: 16pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/analytics.png</normaloff>images/analytics.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRatingSidebar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>160</y>
         <width>100</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(40, 57, 83); 
color: white;
font: 16pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/rating.png</normaloff>images/rating.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btnResourcesSidebar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>240</y>
         <width>100</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(40, 57, 83); 
color: white;
font: 16pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/milk-box.png</normaloff>images/milk-box.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRefillSidebar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>320</y>
         <width>100</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(40, 57, 83); 
color: white;
font: 16pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/coffee-filter.png</normaloff>images/coffee-filter.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="ratingPage" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#centralwidget{
background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(89, 122, 170, 255), stop:1 rgba(255, 255, 255, 255));}</string>
      </property>
      <widget class="QPushButton" name="btnSubmit">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>350</y>
         <width>240</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:15px;
background-color: rgb(40, 57, 83); 
color: white;
font: 14pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string>Submit Rating!</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/rating.png</normaloff>images/rating.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRating1">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>210</y>
         <width>91</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:15px;
background-color: rgb(81, 116, 168);
color: white;
font: 14pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/ratingKaffeebohne.png</normaloff>images/ratingKaffeebohne.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="lblRatingBeanSort">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>130</y>
         <width>500</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Oswald&quot;; color:rgb(0, 0, 0)</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Original Vienna Beans&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblRatinBeans">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>60</y>
         <width>500</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Oswald&quot;; color:rgb(0, 0, 0)</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;You are rating the following beans:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRating2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>210</y>
         <width>91</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:15px;
background-color: rgb(40, 57, 83); 
color: white;
font: 14pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/ratingKaffeebohne.png</normaloff>images/ratingKaffeebohne.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRating4">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>210</y>
         <width>91</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:15px;
background-color: rgb(40, 57, 83); 
color: white;
font: 14pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/ratingKaffeebohne.png</normaloff>images/ratingKaffeebohne.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRating5">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>210</y>
         <width>91</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:15px;
background-color: rgb(40, 57, 83); 
color: white;
font: 14pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/ratingKaffeebohne.png</normaloff>images/ratingKaffeebohne.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRating3">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>210</y>
         <width>91</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:15px;
background-color: rgb(40, 57, 83); 
color: white;
font: 14pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/ratingKaffeebohne.png</normaloff>images/ratingKaffeebohne.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionLogin">
   <property name="icon">
    <iconset>
     <normaloff>images/log-in.png</normaloff>images/log-in.png</iconset>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionSign_up">
   <property name="icon">
    <iconset>
     <normaloff>images/sign-up.png</normaloff>images/sign-up.png</iconset>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </action>
  <action name="actionStats">
   <property name="icon">
    <iconset>
     <normaloff>images/analytics.png</normaloff>images/analytics.png</iconset>
   </property>
   <property name="text">
    <string>Stats</string>
   </property>
  </action>
  <action name="actionResources">
   <property name="icon">
    <iconset>
     <normaloff>images/milk-box.png</normaloff>images/milk-box.png</iconset>
   </property>
   <property name="text">
    <string>Resources</string>
   </property>
  </action>
  <action name="actionProfile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/settings.png</normaloff>images/settings.png</iconset>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionRefill">
   <property name="icon">
    <iconset>
     <normaloff>images/coffee-filter.png</normaloff>images/coffee-filter.png</iconset>
   </property>
   <property name="text">
    <string>Refill</string>
   </property>
  </action>
  <action name="actionRating">
   <property name="icon">
    <iconset>
     <normaloff>images/rating.png</normaloff>images/rating.png</iconset>
   </property>
   <property name="text">
    <string>Rating</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
